generator/src/test/java/com/scottlogic/deg/generator/cucumber/steps/GeneralTestStep.java

```diff
@@ -206,6 +206,17 @@ private GeneratedTestData getExpectedAndGeneratedData(List<Map<String, String>>
         return new GeneratedTestData(expectedRowsOfResults, data);
     }
 
+    @Then("some data should be generated")
+    public void someDataShouldBeGenerated() {
+        List <List<Object>> data = cucumberTestHelper.generateAndGetData();
+        Assert.assertFalse(data.isEmpty());
```

Comment:
I'd suggest including a message here, otherwise it will display 'expected false buy got true'.

Maybe something like, "no data was generated but some was expected"